metadata.rb

```diff
@@ -6,6 +6,6 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url       'https://github.com/aetrion/chef-stunnel'
 issues_url       'https://github.com/aetrion/chef-stunnel/issues'
-version          '2.1.0'
+version          '2.2.0'
```

Comment:
Could you roll back this change? We would prefer to do version bumping manually for releases on master and not in a branch.